lang/en_US.ini

```diff
@@ -297,6 +297,7 @@ enable_auto_save = "Enable Auto Save"
 explain_autosave = "When enabled, new content or draft will be automatically saved every 60 seconds."
 login_protect_system = "Login protection system"
 cloudflare_info = "Review Cloudflare's Turnstile documentation: "
+friendly_captcha_info="Review Friendly Captcha's documentation: "
```

Comment:
I have added to en_US.ini alone so far. Does this needed to be included in all the language files or it happens auto-magically?